share/picongpu/tests/Synchrotron_plugin/include/picongpu/param/density.param

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2013-2023 Axel Huebl, Rene Widera, Felix Schmitt,
+ *                     Richard Pausch, Marco Garten
```

Comment:
```suggestion
/* Copyright 2013-2024 Axel Huebl, Rene Widera, Felix Schmitt,
 *                     Richard Pausch, Marco Garten, Filip Optolowicz
```

Reply:
Most of the license headers are still showing `2013-2023`, I think @psychocoderHPC wanted to update all the headers together